ilastik/applets/pixelClassification/pixelClassificationGui.py

```diff
@@ -493,7 +491,25 @@ def __init__(self, parentApplet, topLevelOperatorView, labelingDrawerUiPath=None
             except:
                 self.render = False
 
-        self.setLiveUpdateEnabled(False)
+        def sync_freeze_value_with_live_update():
+            self._update_freeze_status(not self.isLiveUpdateEnabled())
+
+        self.labelingDrawerUi.liveUpdateButton.toggled.connect(sync_freeze_value_with_live_update)
+
+        self._update_live_update_button(not self.topLevelOperatorView.FreezePredictions.value)
+        # need to initialize the aux buttons once in case _update_live_update_button did not 
+        # result in an update
+        self._update_aux_buttons(not self.topLevelOperatorView.FreezePredictions.value)
+
+        def sync_live_update_with_freeze_value():
+            self._update_live_update_button(not self.topLevelOperatorView.FreezePredictions.value)
+
+        # listen to freezePrediction changes
+        self.topLevelOperatorView.FreezePredictions.notifyDirty(bind(sync_live_update_with_freeze_value))
+        self.__cleanup_fns.append(partial(
```

Comment:
`notifyDirty` already returns an unsubscribe callback, so one could use it directly.